lib/shop_invader/middlewares/erp_proxy.rb

```diff
@@ -0,0 +1,25 @@
+module ShopInvader
+  module Middlewares
+    class ErpProxy < Locomotive::Steam::Middlewares::ThreadSafe
+
+      include Locomotive::Steam::Middlewares::Helpers
+
+      def _call
+        params = env['rack.request.form_hash']
+        puts params && params.include?('action_proxy')
+        if params && params.include?('action_proxy')
+          method = env['REQUEST_METHOD']
+          path = params.delete('action_proxy')
+          erp.call(env['REQUEST_METHOD'], path, params)
```

Comment:
I expect to make a call to the ERP (adding a product to the cart, updating the cart, updating the customer address...)
I do not care of the result as the engine will render the page requested (the page where we have done the action).
For some action like modifing the cart Odoo will ask to shopinvader to store the result (the cart in json) into the session, so this avoid to request odoo when we access to the store.cart

Most of the time I do not care of the result as the render of the page will be render